tensorflow_datasets/core/dataset_builder.py

```diff
@@ -38,6 +38,7 @@
 from tensorflow_datasets.core import splits as splits_lib
 from tensorflow_datasets.core import units
 from tensorflow_datasets.core import utils
+from tensorflow_datasets.core.utils import py_utils
```

Comment:
Actually utils already includes all of py_utils, so you can rm this and do `utils.free_disk_size`

Reply:
Yes I see but I didn't know what to do because I just thought readability becomes poor, I was confused sorry!

Reply:
Fixed it.